lib/Entity/Widget.php

```diff
@@ -1296,6 +1299,9 @@ private function unlinkMedia()
             $i++;
             $sql .= ',:mediaId' . $i;
             $params['mediaId' . $i] = $mediaId;
+
+            // audit the media being deleted
+            $this->getLog()->audit('Media', $mediaId, 'Media Removed', ['mediaId' => $mediaId, 'widgetId' => $this->widgetId]);
```

Comment:
This makes sense - although same as above regarding the message.

It would also be useful to log any mediaId's removed during Widget delete on the existing audit message for widgets that have been deleted.